### DISCUSSION TITLE
BUG: GCP test latency discrepancy between browsers (Chrome result is wrong).

### DISCUSSION THREAD

**Discussion by redbar0n** [VarunAgw/CloudPingTest#36] [Q&A] [answered]
The bug in https://github.com/VarunAgw/CloudPingTest/discussions/32 seems to have reappeared.

But this time not in Hetzner but in the GCP test in Brave (Chrome based):
https://cloudpingtest.com/gcp

In Brave (Chrome based), it gives 3-7 ms for all regions. It seems wrong.

In Firefox, however, it gives 200-300 ms.

**Comment by mnp** [comment]
I'm seeing some impossible numbers to GCP and Azure as well.  Like, low single digit ms across the world.

Firefox 127.

**Reply by leglock** [reply]
I just entered bug #55 which has details specific to what is going on with Azure.

**Reply by ThisCatLikesCrypto** [reply]
are you using uBlock Origin? turning it off on this site fixed it for me

**Comment by leglock** [comment]
These false speeds are happening because the Google run.app domain being pinged is blocking requests from cloudpingtest.com. When I request those same URLs with a different referrer (or no referrer at all) I get a response.

![image](https://github.com/user-attachments/assets/f97328b9-0d66-4a07-8603-722b58736531)

I am going to submit a minor change to the JavaScript that will excludes the referrer from being sent when pinging. I think that will fix this.

**Comment by VarunAgw** [comment, marked as answer]
Working fine for me on Chrome. Is it brave specific issue?

**Reply by ThisCatLikesCrypto** [reply]
it could be an issue on brave bc of the built-in adblock
